pkg/binmanRelease.go

```diff
@@ -22,6 +22,24 @@ var (
 	ErrNoVersionsFound = errors.New("No Versions detected for release")
 )
 
+type NoUpdateError struct {
```

Comment:
I wasn't sure exactly if this was the best place to put error structs. Feel free to let me know if you know of a better place.

Reply:
as good a place as any until a bigger refactor gives error structs a good home